_posts/necessities/2018-02-05-chile-oil.md

```diff
@@ -0,0 +1,35 @@
+---
+layout: post
+title: Chile Oil
+category: necessities
+---
+
+I always have chile oil in a jar in the pantry. My mother makes her own, and my recipe is an adaptation of hers. 
+
+- Start with whole, dried chilis. 
+
+- I live in California, and I prefer Mexican dried chilis which tend to be fresher than the ones found in Chinese grocery stores. Freshness is an interesting concept when it comes to something dried--I mean a chile that has a pliable quality to it still, and it isn't so brittle that it feels like dust waiting to happen in your hand. If you live somewhere with hotter summers than I do, you can grow your own peppers and dry them yourself to make chile oil. 
+
+- Sourcing chilis: I've been on a roll with [Casa Ruiz's arbol chiles](http://casaruizbrand.com/), and I've had good results with [Penzey's Sannam Chilis](https://www.penzeys.com/online-catalog/sanaam-chili-peppers/c-24/p-1256/pd-s) as well, but any dried pepper at all can make a chile oil. 
+
+- Remove stems from dried chiles. While doing this, you can shake out any loose seeds if you'd like a slightly less spicy chile oil. I don't usually bother. 
+
+- Heat a well seasoned cast iron pan on the stove. Do not use any oil. Toast the chiles on both sides until they smell fragrant. Sometimes they blister! That's a good thing! If they burn and blacken, start over, or remove the ones that are burnt. A little bit of burnt pepper is OK, but too much creates an acrid oil that you're not going to enjoy eating. It takes around 2 minutes on each side for the peppers to toast, but watch carefully, peppers vary in toasting time. I usually flip them over individually with chopsticks, a spatula is too coarse of a cooking tool for this job. 
+
+- Remove chiles from the pan as soon as they're toasted. If some are ready sooner than others, remove those first! No rules here except get the chiles toasted and keep 'em from burning. 
+
+- Let the chiles cool a little and then grind them in a cheapo blade grinder. Something like [this](https://www.amazon.com/KRUPS-Electric-Grinder-Stainless-3-Ounce/dp/B00004SPEU/ref=sr_1_4?s=home-garden&ie=UTF8&qid=1517894119&sr=1-4&keywords=spice+grinder) serves nicely if you don't already have one. Of course, you can always use a mortar and pestle but it's really not necessary for this recipe. How fine you grind the peppers is up to you, but enjoy a toothiness to my chile oils--a fine dust is a little too much grinding. 
+
+- Put the ground peppers in a heatproof container (a mason jar is good, a plastic jar is not good. You really don't want this container to break while you're pouring hot oil in. You really, really don't.) that you want to store the chile oil in. 
```

Comment:
You can drop the parens here and move the end of the sentence to to its start: 'Put the ground peppers in a heatproof container that will want to store the chili oil in. While a mason jar is good, a plastic jar is not. ...'

More seriously, I think the heatproof container should actually be a metal mixing bowl or something similar. I believe mason jars are not intended to stand really high temperatures and can sometimes shatter under thermal stress. Our episode a month or so ago with this is probably related.